app.js

```diff
@@ -1,20 +1,18 @@
 require("dotenv").config();
 
 const express = require("express");
+const routes = require("./routes");
 const cors = require("cors");
 const logger = require("morgan");
 const { appDataSource } = require("./models/dataSource");
-const routes = require("./routes");
-const { accessToken } = require("./middleware/auth");
-const { globalHanderError } = require("./middleware/error");
-
+const { globalErrorHandler } = require("./middleware/error");
```

Comment:
```suggestion
const express = require("express");
const cors = require("cors");
const logger = require("morgan");

const routes = require("./routes");
const { appDataSource } = require("./models/dataSource");
const { globalErrorHandler } = require("./middleware/error");
```
suggestion을 참고하셔서 각 종류별 모듈들이 섞여 있는 것을 구분 지어주시고, package로서 install 했던 외부 모듈과 개발자가 작성한 custom 모듈을 구분짓는 줄바꿈을 추가해주세요~